testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/KeycloakQuarkusServerDeployableContainer.java

```diff
@@ -142,17 +163,35 @@ private Process startContainer() throws IOException {
         builder.environment().put("KEYCLOAK_ADMIN_PASSWORD", "admin");
 
         if (restart.compareAndSet(false, true)) {
-            FileUtils.deleteDirectory(configuration.getProvidersPath().resolve("data").toFile());
+            if (SystemUtils.IS_OS_WINDOWS) {
+                deleteTempFilesOnWindows(configuration.getProvidersPath().resolve("data"));
+            } else {
+                FileUtils.deleteDirectory(configuration.getProvidersPath().resolve("data").toFile());
+            }
         }
 
         return builder.start();
     }
 
     private String[] getProcessCommands() {
         List<String> commands = new ArrayList<>();
-        commands.add(getCommand());
-        commands.add("-v");
+
+        if(SystemUtils.IS_OS_WINDOWS) {
+            commands.add(configuration.getProvidersPath().resolve("bin") + File.separator + SCRIPT_CMD_INVOKABLE);
```

Comment:
Doesn't make more sense to have the command resolution within the `getCommand` method and only deal with the `-v` option here?

Note that that method is also used to import realms.

Reply:
can do, sure.